client/state/action-types.js

```diff
@@ -148,6 +148,7 @@ export const HAPPYCHAT_OPEN = 'HAPPYCHAT_OPEN';
 export const HAPPYCHAT_RECEIVE_EVENT = 'HAPPYCHAT_RECEIVE_EVENT';
 export const HAPPYCHAT_TRANSCRIPT_RECEIVE = 'HAPPYCHAT_TRANSCRIPT_RECEIVE';
 export const HAPPYCHAT_TRANSCRIPT_REQUEST = 'HAPPYCHAT_TRANSCRIPT_REQUEST';
+export const HAPPYCHAT_SEND_MESSAGE = 'HAPPYCHAT_SEND_MESSAGE';
```

Comment:
We should keep these actions in the alphabetical order. I know the following `SET` actions has been here before this. Maybe we could move them together? 😃